lib/events/athena/publisher.go

```diff
@@ -94,11 +158,28 @@ func newPublisherFromAthenaConfig(cfg Config) *publisher {
 		so.MaxAttempts = 20
 		so.MaxBackoff = 1 * time.Minute
 	})
-	return NewPublisher(PublisherConfig{
-		TopicARN: cfg.TopicARN,
-		SNSPublisher: sns.NewFromConfig(*cfg.PublisherConsumerAWSConfig, func(o *sns.Options) {
+	hc := awshttp.NewBuildableClient().WithTransportOptions(func(t *http.Transport) {
+		// attempt to aggressively reuse connections for the sake of
+		// avoiding TLS handshakes (the default MaxIdleConnsPerHost is a
+		// pitiful 2)
+		t.MaxIdleConns = 10000
+		t.MaxIdleConnsPerHost = 10000
+	})
+	var messagePublisher messagePublisherFunc
+	if cfg.TopicARN == "" {
+		messagePublisher = SQSPublisherFunc(cfg.QueueURL, sqs.NewFromConfig(*cfg.PublisherConsumerAWSConfig, func(o *sqs.Options) {
```

Comment:
Could you add a comment for this case

Reply:
I think it's clearer now with the explicit check against `"bypass"`.